Plugins/GeoModel/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+include(FetchContent)
+
+add_library(
+  ActsPluginGeoModel SHARED
+  src/GeoModelReader.cpp)
+target_include_directories(
+    ActsPluginGeoModel
+  PUBLIC
+    $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include>
+    $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+target_include_directories(
+      ActsPluginGeoModel
+      SYSTEM PUBLIC)    
```

Comment:
Does this work without any include directories?

Reply:
Removed this line, it's not needed.